Back-End/api/utils/utils.py

```diff
@@ -0,0 +1,13 @@
+import requests
```

Comment:
I can make a pretty good guess at what this file does but please give it a slightly more informative name than utils.py as the name utils should really only be reserved for a set of hyper specific user defined functions typically useless outside of that project. For example, a StringUtils.py file would be good candidate if you needed some weird string manipulation in your project.